ThreeBotPackages/threefold/3bot_network_relay/start.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
```

Comment:
This file should be replaced with a package file. See https://github.com/threefoldtech/jumpscaleX_threebot/blob/development/ThreeBotPackages/threefold/phonebook/package.py for example.

Reply:
I don't think this is supposed to run inside a 3bot but rather in one or more central places...